README.md

```diff
@@ -1,4 +1,112 @@
-## Installation
+# Pulldasher
+Pulldasher is self-hosted to-do list for GitHub repositories.
+Pulldasher tracks your pull requests and displays them according to their
+current state. It sports a flexible template system, allowing you to customize
+it extensively without touching the core code.
+
+![pulldasher-image](https://cloud.githubusercontent.com/assets/2539016/11315808/78af398e-8fad-11e5-81d4-b59ae6109dec.png)
+
+Pulldasher is written primarily in JavaScript, using Node.js. See the
+[`package.json`](package.json/) and [`bower.json`](bower.json/) files for more on the front- and back-end
+dependencies, respectively.
+
+To run Pulldasher, you'll need MySQL as well as Node. MySQL is used for
+statistics-gathering and some sorting and filtering.
+
+## Getting Started
 1. `make`
-2. `cp config.example.js config.js; vim config.js`
-3. `bin/pulldasher`
+2. `cp config.example.js config.js; $EDITOR config.js`
+3. Create a new MySQL database and source the schema.sql file.
+ * `create database pulldasher;`
+ * `use pulldasher;`
+ * `source schema.sql;`
+4. `bin/pulldasher`
+
+## Use
+Pulldasher is driven by tags in pull requests and pull comments. Normally, it
+assumes that two code review and one quality assurance signoff will be
+required per pull. This can be adjusted on a per-pull basis by using the
+`cr_req` and `qa_req` tags in a pull description. For example, if you write a
+pull that touches some really dangerous code, you might add the `cr_req 3` and
+`qa_req 2` tags to it, requiring three CR signoffs and two QA signoffs before
+the pull is considered ready. Conversely, if a pull only touches test code, you
+might put only `qa_req 0` on it to say that it doesn't need to be QAed, since
+it should break tests if there's anything wrong with it.
+
+When you CR a pull, add a comment to it with `CR :emoji:` in it.
+(`emoji` is simply a word or words between colons; we often use GitHub emoji,
+which follow this format.) This comment is considered a _signoff_. Pulldasher
+will update the pull's display to indicate that one of the required CRs is
+completed.  Similarly, when you QA a pull, add a comment containing
+`QA :emoji:`, and the number of QA signoffs will increase.
+
+## Magic Features
+There's a couple features which aren't exposed clearly through the UI:
+
+#### Hover Copy
+If you hover over a pull and type `ctrl-c` (`command-c` on a Mac), the branch
+name of the pull will be copied to your clipboard.
+
+#### Filter parameters
+Two query string parameters are available to filter the displayed pulls:
+
+1. `assigned`: Providing a comma-separated list of usernames to the `assigned`
+   parameter will filter the pulls to only those assigned to those users.
+
+2. `milestone`: Providing a comma-separated list of milestones to the
+   `milestone` parameter will filter the pulls to only those on the specified
+   milestones.
```

Comment:
I'd like that. I just tested out using one of our milestone names in the query param and everything seemed to work. I'm not really sure what the rules are on characters that can go in the parameters part of the URL.

[RFC 3986](https://tools.ietf.org/html/rfc3986#section-3.4) says '/'s are fine in a GET.